data/AndroGari.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Garima Saroj",
+  "tag": "SIH'22 Winner | Ex Jr. ML Engineer @JupiterAILabs | CSE(AI&ML) '24"
```

Comment:
As far as I know, there is no valid key named `tag`. Please remove this from your `json`

You can add this info to your bio if you want. 

```suggestion
```